operator/v1/0000_70_cluster-network-operator_01.crd.yaml

```diff
@@ -290,9 +290,23 @@ spec:
                         type: integer
                     type: object
                   ovnKubernetesConfig:
-                    description: oVNKubernetesConfig configures the ovn-kubernetes
-                      plugin. This is currently not implemented.
+                    description: ovnKubernetesConfig configures the ovn-kubernetes
+                      plugin.
                     properties:
+                      gatewayConfig:
+                        description: gatewayConfig holds the configuration for node
+                          gateway options. Supported from OCP 4.10.
```

Comment:
I don't think we usually bake the supported version info into the api definition.

Reply:
ack that does sound like something I shouldn't be doing and probably provides no value to users. Let me remove this.